standalone/src/models/shipping-method/shipping-rate/shipping-rate-price-tier/cart-score/fields-config.ts

```diff
@@ -0,0 +1,30 @@
+import { fake, type TModelFieldsConfig } from '@/core';
+import { MoneyRest } from '@/models/commons';
+import type {
+  TCartScorePriceTierGraphql,
+  TCartScorePriceTierRest,
+} from './types';
+
+// https://docs.commercetools.com/api/projects/shippingMethods#cartscoretier
+
+const commonFieldsConfig = {
+  type: 'CartScore',
+  price: fake(() => MoneyRest.random()),
```

Comment:
We should only populate required fields and, according to the docs, `price` is optional